airbyte-webapp/src/hooks/services/Experiment/experiments.ts

```diff
@@ -26,4 +26,5 @@ export interface Experiments {
   "connection.newTableDesign": boolean;
   "workspace.freeConnectorsProgram.visible": boolean;
   "connectorBuilder.showNavigationLinks": boolean;
+  "billingPage.billingInsights": boolean;
```

Comment:
nitpick:
```suggestion
  "billing.billingInsights": boolean;
```

I know `authPage` follows a "page-first" approach, but I feel like targeting experiments by domain is more flexible. For example, if we want to toggle something in the sidebar or connection page related to this feature, it's not strictly on the `billingPage`.

Reply:
Sure!  It looks like I may not have permissions to archive the flag that exists, though.